settings/common.py

```diff
@@ -333,3 +337,26 @@
 EKS_NODEGROUP_ROLE_ARN = os.environ.get("EKS_NODEGROUP_ROLE_ARN")
 
 ENVIRONMENT = os.environ.get("ENVIRONMENT", "dev")
+
+SIMPLE_JWT = {
+    "ACCESS_TOKEN_LIFETIME": timedelta(days=365),
+    "REFRESH_TOKEN_LIFETIME": timedelta(days=365),
+    "ROTATE_REFRESH_TOKENS": False,
+    "UPDATE_LAST_LOGIN": False,
+
+    "ALGORITHM": "HS256",
+    "SIGNING_KEY": "test-key",
+    "VERIFYING_KEY": None,
```

Comment:
Why is this set as "None"?

Reply:
As we are using HMAC as the encryption algorithm value of `VERIFYING_KEY` will be same as `SIGNING_KEY`. See this https://django-rest-framework-simplejwt.readthedocs.io/en/latest/settings.html#verifying-key